python/snips_nlu_parsers/builtin_entities.py

```diff
@@ -196,3 +198,32 @@ def get_builtin_entity_examples(builtin_entity_kind, language):
             _ENTITIES_EXAMPLES[builtin_entity_kind][language] = list(
                 array.data[i].decode("utf8") for i in range(array.size))
     return _ENTITIES_EXAMPLES[builtin_entity_kind][language]
+
+
+def get_complete_entity_ontology_json():
+    """Lists the complete entity ontology for all languages in JSON format
+    """
+    global _COMPLETE_ENTITY_ONTOLOGY_JSON
+    if _COMPLETE_ENTITY_ONTOLOGY_JSON is None:
+        with string_pointer(c_char_p()) as ptr:
+            exit_code = lib.snips_nlu_ontology_complete_entity_ontology_json(byref(ptr))
+            check_ffi_error(exit_code, "Something went wrong when retrieving "
+                                       "complete entity ontology")
+            result = string_at(ptr)
+            _COMPLETE_ENTITY_ONTOLOGY_JSON = result.decode("utf8")
+    return _COMPLETE_ENTITY_ONTOLOGY_JSON
+
+
+def get_language_entity_ontology_json(language):
+    """Lists the complete entity ontology for the specified language in JSON format
+    """
+    global _LANGUAGE_ENTITY_ONTOLOGY_JSON
+    if language not in _LANGUAGE_ENTITY_ONTOLOGY_JSON:
+        with string_pointer(c_char_p()) as ptr:
+            exit_code = lib.snips_nlu_ontology_language_entity_ontology_json(
+                language.encode("utf8"), byref(ptr))
+            check_ffi_error(exit_code, "Something went wrong when retrieving "
+                                       "language entity ontology")
+            result = string_at(ptr)
+            _LANGUAGE_ENTITY_ONTOLOGY_JSON[language] = result.decode("utf8")
+    return _LANGUAGE_ENTITY_ONTOLOGY_JSON[language]
```

Comment:
As we do not really need the raw JSON string, the json loading should probably be done inside these two functions, which should be renamed without the `_json` suffix.

Reply:
k